book/src/configuration.md

```diff
@@ -46,6 +46,7 @@ hidden = false
 | `auto-info` | Whether to display infoboxes | `true` |
 | `true-color` | Set to `true` to override automatic detection of terminal truecolor support in the event of a false negative. | `false` |
 | `rulers` | List of column positions at which to display the rulers. Can be overridden by language specific `rulers` in `languages.toml` file. | `[]` |
+| `read-only-indicator` | String to display next to filename in statusbar when configured and if file is read-only. Set to `""` to disable. | `"[RO]"` |
 | `color-modes` | Whether to color the mode indicator with different colors depending on the mode itself | `false` |
```

Comment:
A tiny improvement idea: I think it might be better to have both `read-only-indicator` _and_ `color-modes` under `[editor.statusline]`, because they're specific to the statusline (and I can't imagine that changing in the future).

Now, I know that in master, `color_modes` was there already. Mostly because when it was introduced by #2676, the statusline PR #2434 wasn't merged yet.

But I don't think it's too late for a little improvement which could make the configuration a bit more intuitive.

Reply:
I was also hoping to add the RO indicator to the bufferline when it gets merged :)

Reply:
Right, didn't think about that one! In that case, you can disregard my suggestion. :)